packages/sdk/src/web/experimental/index.ts

```
@@ -21,24 +21,22 @@ export async function init<T extends SchemaType>(options: SDKConfig) {
         ...options.client,
     } as torii.ClientConfig;
 
-    const client = await torii.createClient(clientConfig);
+    const client = new torii.ToriiClient(clientConfig);
 
     return {
         getEntities: async (query: torii.Query) => {
-            return parseEntities(await client.getEntities(query, false));
+            return parseEntities((await client.getEntities(query)).items);
         },
         getEvents: async (query: torii.Query, historical: boolean = false) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)